### PR TITLE
feat:added profile details and leaderboard

### PR DESCRIPTION
**Pull request by poornima-alugubelly**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+	--primary-color: #0b6875;
+	--bg-color: #ededed;
+	--secondary-color: #181919;
+	--ternary-color: rgb(251 254 255);
+	--shadow-color: rgb(176 196 208 / 40%);
+	--primary-dark: rgb(71 130 123);
+	--ternary-dark: rgb(201 214 213);
+	--dark-blue: rgb(0, 0, 0, 0.5);
+	--black: rgb(219 230 230);
+	--scroll-color: #07464e;
```
Comment (msdeshmukh009) [inline src/App.css:13]: Not a major thing. But try to use the same color model for all your colors.

File: src/components/NavBar.tsx [hunk 2]
```diff
@@ -1,27 +1,61 @@
...
+						<i className="fas fa-trophy btn-icon "></i>
+					</Link>
+					<div>
+						{theme === "lightQuiz" ? (
+							<div className="flex-column flex-align-center gap-xs">
+								<i
+									className="far fa-moon btn-icon"
+									onClick={() => {
+										setTheme("");
+										localStorage.setItem("theme", "");
```
Comment (msdeshmukh009) [inline src/components/NavBar.tsx:28]: We can extract this function.

**Review by preethasri — APPROVED** [review]
good to merge

**Review by Varu98 — APPROVED** [review]
looks good to merge.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

**Pull request merged**